### ISSUE TITLE
_typed_dict_test_helper.py is missing from the PyPI archive

### ISSUE THREAD

**Issue by kulikjak** [python/typing_extensions#61]
Hi, the archive downloaded from PyPI does include `test_typing_extensions.py`, but it omits `_typed_dict_test_helper.py` meaning that  running `pytest test_typing_extensions.py` results in:
```
ImportError while importing test module 'xxxx/typing_extensions-4.3.0/src/test_typing_extensions.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/lib/python3.7/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_typing_extensions.py:32: in <module>
    from _typed_dict_test_helper import FooGeneric
E   ModuleNotFoundError: No module named '_typed_dict_test_helper'
```

**Comment by srittau** [comment]
Thank you for the report, I think I spotted the error in our `pyproject.toml` file.

**Comment by srittau** [comment]
Actually when I run `flit build` locally I get totally different files than what's on PyPI. (And definitely too many, like the whole `.github` directory.) Cc @JelleZijlstra, who did the last release and might have more experience with flit.

**Comment by hegjon** [comment]
Fixed in #97

**Comment by AlexWaygood** [comment]
> Fixed in #97

Thanks!
